qemu/tests/cfg/device_assignment.cfg

```diff
@@ -7,7 +7,19 @@
     variants:
         - multi_pf_boot:
             type = boot
-            nics = "nic1 nic2"
+            nics = "nic1 nic2 nic3"
+            pci_assignable_nic1 = no
+            nic_model_nic1 = virtio
+            driver_option = 0
+            variants:
+                - 82576:
+                    pf_filter_re = 82576
+                - XL710:
+                    pf_filter_re = XL710
+                - 82599:
+                    pf_filter_re = 82599
+                - BCM57810:
+                    pf_filter_re = BCM57810
```

Comment:
@luckyh @yanan-fu , I would prefer to have variants in test script specific cfg (like sriov.cfg) with different test scenarios instead of same test with different cards. I agree that we need less manual configuration for which I would have the consolidated test cases in cfg which will be given to --vt-config. so the different cards can be added as variants in this cfg with the test case.